polaris-migrator/src/migrations/replace-spacing-lengths/replace-spacing-lengths.ts

```diff
@@ -10,35 +8,17 @@ import {
   isSassFunction,
   isTransformableLength,
   namespace,
-  NamespaceOptions,
   toTransformablePx,
   StopWalkingFunctionNodes,
+  createSassMigrator,
 } from '../../utilities/sass';
 import {isKeyOf} from '../../utilities/type-guards';
 
-export default function replaceSpacingLengths(
-  fileInfo: FileInfo,
-  _: API,
-  options: Options,
-) {
-  return postcss(plugin(options)).process(fileInfo.source, {
-    syntax: require('postcss-scss'),
-  }).css;
-}
-
-const processed = Symbol('processed');
-
-interface PluginOptions extends Options, NamespaceOptions {}
-
-const plugin = (options: PluginOptions = {}): Plugin => {
+export default createSassMigrator('replace-sass-space', (options) => {
   const namespacedRem = namespace('rem', options);
 
   return {
-    postcssPlugin: 'replace-sass-space',
     Declaration(decl) {
-      // @ts-expect-error - Skip if processed so we don't process it again
-      if (decl[processed]) return;
```

Comment:
So it turns out this migration had a but in it 😱 

While we were checking for `decl[processed]` here, we were never actually setting it!

The `createSassMigrator()` abstracts that logic away, so individual migrations can't make that same mistake again 👍